tutor/src/components/cc-dashboard/section-performance.cjsx

```diff
@@ -29,9 +39,10 @@ SectionPerformance = React.createClass
 
     <div>
       <BS.ProgressBar className="reading-progress-group">
-        {correctBar}
-        {incorrectBar}
+        {bars}
       </BS.ProgressBar>
```

Comment:
This is the actual bug - if either `correctBar` or `incorrectBar` is null, the `BS.ProgressBar` component references something it should.